guide/english/computer-science/threads/index.md

```diff
@@ -32,6 +32,35 @@ if __name__ == "__main__":
     for i in range(1, 11):
         threading.Thread(target=print_number, args=(i,)).start()
 ````
+Java Implementation
```

Comment:
Add heading indentation to this to make it stand out and that the reader will be able to differentiate it.

tip: for adding a heading in GitHub markdown, simply precede the line with a hash `#` with number of hashes symbolising type of heading. headings work just like html. 
e.g

`h1: # `
`h2: ##`
`h3: ###`